### DISCUSSION TITLE
Is it possible to create nested routes in compatibility mode

### DISCUSSION THREAD

**Discussion by wbolduc** [remix-run/react-router#9979] [v5 to v6 Migration]
https://codesandbox.io/s/nested-routes-dont-work-in-compatibility-mode-flwz7p

In this sandbox, unless I'm doing it wrong, it appears that it's not possible to render nested routes with CompatRoute

Is this possible?

I would like to be able to do this because some pages in our v5 version would be much better suited as nested routes

**Comment by Savinien92** [comment]
Any updates about this topic ?
